crates/octez/src/endpoint.rs

```diff
@@ -16,6 +16,16 @@ impl Endpoint {
             port,
         }
     }
+
+    pub fn get_authority(&self) -> String {
```

Comment:
nit:
`get_` isn't used in rust convention IIRC, what about `to_authority_string`?